src/svd.jl

```diff
@@ -9,7 +9,19 @@ struct SVD{T,TU,TS,TVt} <: Factorization{T}
 end
 SVD(U::AbstractArray{T}, S::AbstractVector, Vt::AbstractArray{T}) where {T} = SVD{T,typeof(U),typeof(S),typeof(Vt)}(U, S, Vt)
 
-getindex(::SVD, ::Symbol) = error("In order to avoid type instability, StaticArrays.SVD doesn't support indexing the output of svdfact with a symbol.  Instead, you can access the fields of the factorization directly as f.U, f.S, and f.Vt")
+function Base.getproperty(F::SVD, s::Symbol)
```

Comment:
`@inline` to guarantee constant propagation of `s`? (The `'` below could expand to be a large amount of code, preventing inlining, yet we need this function to be type-inferred)